src/vl-wizard-pane.js

```diff
@@ -245,25 +245,28 @@ export class VlWizardPane extends vlElement(HTMLElement) {
     const previousAction = this._previousAction;
     if (previousAction) {
       this._previousAction.addEventListener('click', () => {
-                this.isPreviousPaneDisabled ? this.disablePreviousPane() : this.enablePreviousPane();
+        this.isPreviousPaneDisabled ? this.disablePreviousPane() : this.enablePreviousPane();
       });
     }
     if (nextAction) {
       this._nextAction.addEventListener('click', () => {
-                this.isNextPaneDisabled ? this.disableNextPane() : this.enableNextPane();
+        this.isNextPaneDisabled ? this.disableNextPane() : this.enableNextPane();
       });
     }
   }
 
   _observeActiveClass(callback) {
     const observer = new MutationObserver((mutations) => {
-      const wasActive = (mutation) => this._isActive(mutation.oldValue ? mutation.oldValue.split(' ') : []);
-      const isActive = (mutation) => mutation.target.isActive;
-      if (mutations.some((mutation) => !wasActive(mutation) && isActive(mutation))) {
-        callback();
+      if (this.isActive) {
+        if (!this.__active && mutations.some((mutation) => mutation.target.isActive)) {
```

Comment:
De "if(this.isActive)" en de "if(!this.__active)" lezen wel wat gek. Mijn eerste gedacht is dan "hoe kan isActive true zijn maar __active false".

Reply:
Misschien beter `__active` hernoemen naar `__wasActive`? Verduidelijkt dat de code?

Reply:
Dat geeft de bedoeling/betekenis inderdaad al wat beter weer